apps/src/storage/dataBrowser/table-controls.module.scss

```diff
@@ -0,0 +1,32 @@
+.buttonWrapper {
+    display: inline-block;
+    margin-bottom: 10px;
+    margin-top: 10px;
+};
+.container {
+    // subtract the height of the clearfix element
+    margin-bottom: -28px;
+    // subtract the top margin of the buttonWrapper
+    margin-top: -10px;
+    padding-top: 0px;
+    padding-bottom: 10px;
+    padding-left: 0px;
+    padding-right: 0px;
+    // make the buttons align right usually, but align left if they
+    // are forced to wrap onto the next line by a very long table name.
+    text-align: justify
```

Comment:
need a `;` at the end of this rule

Reply:
I am seeing a few others as well. I'll do a thorough pass now.